### DISCUSSION TITLE
Can't register

### DISCUSSION THREAD

**Discussion by islam-mdshehab** [kyujin-cho/pixel-volte-patch#65] [General]
I followed your steps but can't register IMS. You said to reboot couple time.. every time I reboot, the shizuku app stops running... Is it normal? 
Do I have to start running shizuku every time I reboot until the IMS showing registered?? Or just rebooting as it is ( without starting shizuku everytime)... Please tell

**Comment by godelosane** [comment]
Hi, could you solve it? I have exactly the same problem.
